src/util.js

```diff
@@ -44,6 +45,7 @@ export function convertToSlipData(source = {}, intl, timeZone, locale, slipName
     'requester.firstName': requester.firstName,
     'requester.lastName': requester.lastName,
     'requester.middleName': requester.middleName,
+    'requester.preferredFirstName': requester.preferredFirstName ? requester.preferredFirstName : requester.firstName,
```

Comment:
Could you please investigate can we change to requester.preferredFirstName || requester.firstName,?

Reply:
This change is resulting in some unit test failure. I think it is fine to retain the current way